oak_crypto/proto/crypto.proto

```diff
@@ -21,6 +21,26 @@ package oak.crypto;
 option java_multiple_files = true;
 option java_package = "oak.crypto";
 
+// Request message encrypted using Hybrid Public Key Encryption (HPKE).
+// <https://www.rfc-editor.org/rfc/rfc9180.html>
+message HpkeRequest {
+  // Message encrypted with Authenticated Encryption with Associated Data (AEAD)
+  // using the derived session key.
+  AeadEncryptedMessage encrypted_message = 1;
+  // Ephemeral Diffie-Hellman key that is needed to derive a session key.
+  // Only sent in the first message of the secure session.
+  optional bytes serialized_encapsulated_public_key = 2;
```

Comment:
nit: This should probably be "encapsulation" rather than "encapsulated", as it is the public key that is used for the Diffie-Hellman encapsulation. The AEAD key is the "encapsulated" key.

Reply:
RFC calls them `encapsulated keys`: https://www.rfc-editor.org/rfc/rfc9180.html#name-dh-based-kem-dhkem